## E1140 (unhashable-dict-key)

### :x: Problematic code:

```python
numbers = [1, 2, 3]
data[numbers] = len(numbers)
print(data)
```

### :heavy_check_mark: Correct code:

```python
numbers = (1, 2, 3)
data[numbers] = len(numbers)
print(data)
```

### Rationale:

Emitted when a dict key is not hashable (i.e. doesn't define __hash__
method).

### Related resources:

- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22unhashable-dict-key%22+OR+%22E1140%22)
